cmd/start.go

```diff
@@ -42,25 +43,34 @@ var startCmd = &cobra.Command{
 
 func init() {
 	LostromosCmd.AddCommand(startCmd)
-	startCmd.Flags().String("kube-config", filepath.Join(homeDir(), ".kube", "config"), "absolute path to the kubeconfig file. Only required if running outside-of-cluster.")
+
 	startCmd.Flags().String("crd-name", "", "the plural name of the CRD you want monitored (ex: users)")
 	startCmd.Flags().String("crd-group", "", "the group of the CRD you want monitored (ex: stable.wpengine.io)")
 	startCmd.Flags().String("crd-version", "v1", "the version of the CRD you want monitored")
 	startCmd.Flags().String("crd-namespace", metav1.NamespaceNone, "(optional) the namespace of the CRD you want monitored, only needed for namespaced CRDs (ex: default)")
-	startCmd.Flags().String("templates", "", "absolute path to the directory with your template files")
+	startCmd.Flags().String("helm-chart", "", "Path for helm chart")
+	startCmd.Flags().String("helm-ns", "", "Namespace for resources deployed by helm")
+	startCmd.Flags().String("helm-prefix", "", "Prefix for release names in helm")
+	startCmd.Flags().String("helm-tiller", "", "Address for helm tiller (ex: 127.0.0.1:44134)")
```

Comment:
Would it make sense to default this to 'tiller-deploy.kube-system' since thats where the tiller is normally deployed?

Reply:
I have a hard time with this one. I like sane defaults but I don't know what the best option for this one is. Maybe just `tiller-deploy` so it talks to the one in the same namespace?